types/sharp-timer/sharp-timer-tests.ts

```diff
@@ -1,7 +1,11 @@
-import { Timer, Stopwatch } from 'sharp-timer';
+import { Timer, Stopwatch, Timespan } from 'sharp-timer';
+import * as s from 'sharp-timer';
+import S = require('sharp-timer');
```

Comment:
why the triplicate import? There's no export=, so the last import shouldn't even work. And there are no examples of usage with the second line.